src/main/java/io/quarkus/search/app/hibernate/InputProviderHtmlBodyTextBridge.java

```diff
@@ -21,13 +21,13 @@ public String toIndexedValue(InputProvider provider, ValueBridgeToIndexedValueCo
             // Content div has two grid columns: actual content and TOC. There's not much use of the TOC, we want the content only:
             Element content = body.selectFirst(".guide");
             if (content != null) {
-                // We may be looking at a guide with/without a TOC;
-                // if it is one with the TOC, there's no uch point in indexing the TOC itself, hence we don't include it:
-
-                // column (grid items) are not present in a guide like all-config
-                Element guideColumn = content.selectFirst(".grid__item");
-                if (guideColumn != null) {
-                    content = guideColumn;
+                Element toc = content.selectFirst(".tocwrapper");
```

Comment:
hmm I do not see this class in the docs... is it something new to be deployed ? for the current quarkus.io I can only find `.toc` class

Reply:
I see it... ?

![image](https://github.com/quarkusio/search.quarkus.io/assets/412878/58fea517-2d27-4ee5-a724-76930771c023)

Reply:
Anyway, with your new approach I can just remove both `.tocwrapper` and `.toc`. That should be more future-proof.

Reply:
hmm, ok 😃 I see it now, too. Not sure what page I was looking at or maybe I should put some 👓  🤷🏻‍♂️